README.md

```diff
@@ -139,11 +139,11 @@ highlight clear LspWarningLine
 #### Highlights
 
 Highlighting diagnostics requires either NeoVim 0.3+ or Vim with patch 8.1.0579.
-They are enabled by default when supported, but can be turned off respectively by
+They are not enabled by default and can be turned on respectively by
 
 ```viml
-let g:lsp_highlights_enabled = 0
-let g:lsp_textprop_enabled = 0
+let g:lsp_highlights_enabled = 1
```

Comment:
Shouldn't it be line 171 that should be changed?

Reply:
Fixed. Might be we need to start removing these docs from README.md and tell people to refer to docs. README.md was used before since we didn't have docs. README should most likely be changed so new people can have a look at it.